components/grid/row.tsx

```diff
@@ -83,14 +101,53 @@ const Row = React.forwardRef<HTMLDivElement, RowProps>((props, ref) => {
     return results;
   };
 
+  // ================================== calc reponsive data ==================================
+  const clacMergeAlign = () => {
+    if (typeof align === 'object') {
+      for (let i = 0; i < responsiveArray.length; i++) {
+        const breakpoint: Breakpoint = responsiveArray[i];
+        // When 'align' sets the 'other' attribute,
+        // we need to set the value of the response attribute not explicitly set in 'align' to the value of 'other'
+        const curAlign = align[breakpoint];
+        if (align.other && !curScreens[breakpoint]) {
+          const otherVal = align.other;
+          if (!align[breakpoint]) {
+            setMergeAlign(otherVal);
+          }
+        } else if (curScreens[breakpoint] && curAlign !== undefined) {
+          setMergeAlign(curAlign!);
+        }
+      }
+    }
+  };
+
+  const clacMergeJustify = () => {
+    if (typeof justify === 'object') {
```

Comment:
```suggestion
```suggestion
    if (typeof align !== 'object') {
      return;
    }
```
    
```

Reply:
done